src/components/AnimatedSleepingSheep/AnimatedSleepingSheep.vue

```diff
@@ -0,0 +1,53 @@
+    <path fill-rule="evenodd" clip-rule="evenodd" d="M74.5424,126.469L68.6131,120.539C67.8372,119.763,66.5782,119.762,65.8009,120.539C65.025,121.317,65.025,122.576,65.8009,123.351L71.7302,129.281C72.5075,130.057,73.7665,130.057,74.5424,129.281C75.3197,128.505,75.3183,127.244,74.5424,126.469Z" fill="#19194B" transform="translate(2.623,44.9999)"/>
+    <path fill-rule="evenodd" clip-rule="evenodd" d="M59.8704,117.421C60.6476,118.197,61.9066,118.197,62.6826,117.421C63.4598,116.645,63.4585,115.385,62.6826,114.609L56.7533,108.68C55.9773,107.904,54.7183,107.902,53.9411,108.68C53.1638,109.457,53.1652,110.716,53.9411,111.492L59.8704,117.421Z" fill="#19194B" transform="translate(2.623,44.9999)"/>
+    <path id="Boca"
+          fill-rule="evenodd"
+          clip-rule="evenodd"
+          d="M57.3772,138.856C54.0158,138.856,51.2914,141.58,51.2914,144.94C51.2914,148.301,54.0158,151.026,57.3772,151.026C60.7372,151.026,63.4617,148.301,63.4617,144.94C63.4617,141.58,60.7372,138.856,57.3772,138.856Z"
+          fill="#19194B"
+          transform="translate(2.623,44.9999)"
+          style="animation: 1.9s linear infinite both Boca_d;"
+    />
+  </svg>
+</template>
+<style>
+  @keyframes a0_t { 0% { transform: translate(10.9521px,63.8029px); } 11.8421% { transform: translate(17.925956px,57.421886px); } 23.6842% { transform: translate(21.468963px,48.846615px); } 35.5263% { transform: translate(21.273475px,39.393112px); } 47.3684% { transform: translate(18.7792px,30.4297px); } 52.6315% { transform: translate(17.9128px,27.014px); } 61.8421% { transform: translate(17.443412px,20.511124px); } 71.0526% { transform: translate(18.240142px,14.026737px); } 80.2631% { transform: translate(20.473189px,7.893995px); } 89.4736% { transform: translate(24.5246px,2.7799px); } 100% { transform: translate(24.5246px,2.7799px); } }
```

Comment:
This kind of identifier should be uniques, adding a prefix could be a solution, like:
```suggestion
  @keyframes animated-sleeping-sheep-a0_t { 0% { transform: translate(10.9521px,63.8029px); } 11.8421% { transform: translate(17.925956px,57.421886px); } 23.6842% { transform: translate(21.468963px,48.846615px); } 35.5263% { transform: translate(21.273475px,39.393112px); } 47.3684% { transform: translate(18.7792px,30.4297px); } 52.6315% { transform: translate(17.9128px,27.014px); } 61.8421% { transform: translate(17.443412px,20.511124px); } 71.0526% { transform: translate(18.240142px,14.026737px); } 80.2631% { transform: translate(20.473189px,7.893995px); } 89.4736% { transform: translate(24.5246px,2.7799px); } 100% { transform: translate(24.5246px,2.7799px); } }
```

Reply:
Thanks! nice catch.. however they are uniques as they are, right? anyway I like to add something related with the component name instead! 👌